pkg/apis/apps/v1alpha1/apimanagerrestore_types.go

```diff
@@ -30,6 +30,10 @@ type APIManagerRestoreStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
+
+	// +optional
+	APIManagerToRestoreRef *v1.LocalObjectReference `json:"apimanagerToRestore,omitempty"`
```

Comment:
An alternative to LocalObjectReference would be ObjectReference, which has the namespace field too that we could set. The thing with ObjectReference is that it also has a bunch of other fields like 'UID types.UID', 'resourceVersion', ...
What do you think? do you think LocalObjectReference good enough?